gtsfm/evaluation/compare_colmap_outputs.py

```diff
@@ -107,6 +116,9 @@ def compare_poses(baseline_dirpath: str, eval_dirpath: str, output_dirpath: str)
         help="Path to directory containing benchmark artifacts for the current",
     )
     parser.add_argument("--output", required=True, help="Output for the json file for pose metrics")
+    parser.add_argument(
+        "--use_pycolmap_alignment", action="store_true", help="Use Pycolmap to align cameras between two reconstruction"
```

Comment:
intended to be false by default? which is superior?

Reply:
Our robust method seems to perform better, but it is slow.